tests/Behat/Element/Admin/Channel/ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Element\Admin\Channel;
+
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElement extends Element implements ShowTheLowestPriceOfDiscountedProductsPriorToTheDiscountElementInterface
+{
+    private const FIELD_LABEL = 'Show the lowest price of discounted products prior to the current discount?';
+
+    public function enable(): void
+    {
+        $this->getDocument()->checkField(self::FIELD_LABEL);
+    }
+
+    public function disable(): void
+    {
+        $this->getDocument()->uncheckField(self::FIELD_LABEL);
+    }
+
+    public function isEnabled(): bool
+    {
+        return $this->getDocument()->hasCheckedField(self::FIELD_LABEL);
+    }
```

Comment:
Create a named element pointing at the field's name/id instead of relying on a label